src/workerd/api/http.c++

```diff
@@ -1195,6 +1196,22 @@ void Request::shallowCopyHeadersTo(kj::HttpHeaders& out) {
 }
 
 kj::Maybe<kj::String> Request::serializeCfBlobJson(jsg::Lock& js) {
+  if(cacheMode != CacheMode::NONE &&
+    util::Autogate::isEnabled(util::AutogateKey::HTTP_REQUEST_CACHE)) {
+    // Works because of the fact there are only two non-none cache modes
+    auto ttl = (CacheMode::NOSTORE == cacheMode) ? -1 : 0;
+    CfProperty clone;
+    KJ_IF_SOME(_, cf.get(js)) {
+      clone = cf.deepClone(js);
+    } else {
+      clone = CfProperty(js, js.obj());
+    }
+    auto obj = KJ_ASSERT_NONNULL(clone.get(js));
+
+    obj.set(js, "cacheTtl", js.num(ttl));
```

Comment:
What does it mean to set `cacheTtl` to `-1`? Documentation for `cacheTtl` says:

> Force response to be cached for a given number of seconds. (e.g. 300)

So I'd expect it to be nonnegative? What is the expected behavior difference between `NOSTORE` and `NOCACHE`?

Reply:
cacheTtl: -1 internally means to bypass cache.

Reply:
Could you update the documentation for `cacheTtl` to include that?